.github/workflows/ci.yml

```diff
@@ -15,6 +15,6 @@ jobs:
           cache: 'yarn'
           cache-dependency-path: yarn.lock
       - run: yarn install --frozen-lockfile
+      - run: yarn workspaces run prepack
```

Comment:
please comment on why prepacking before lint. it really shouldn't be necessary before `test` so consider,
```
test
# prepack to the linter has access to the build exports of other packages
prepack
lint
```

Reply:
It was failing if not prepacked before `test` with this error: https://github.com/Agoric/ui-kit/actions/runs/8069865129/job/22045857705

Just added a comment.